doc/nrf/ug_nrf52_developing.rst

```diff
@@ -90,6 +90,21 @@ Consider using these features in your project to speed up the FOTA upgrade proce
 
 .. fota_upgrades_end
 
+.. fota_upgrades_bt_mesh_start
+
+FOTA in Bluetooth mesh
```

Comment:
```suggestion
FOTA in Bluetooth® mesh
```
I am not certain if the copy right should be added to Bluetooth here and other places.